sql/bduni_convert.sql

```diff
@@ -286,35 +290,35 @@ INSERT INTO edges
 -- ############################
 
 -- On ne conserve que les non communications sur la zone de calcul
-DROP TABLE IF EXISTS non_comm;
-CREATE TABLE IF NOT EXISTS non_comm AS
+DROP TABLE IF EXISTS {output_schema}.non_comm;
+CREATE TABLE IF NOT EXISTS {output_schema}.non_comm AS
 SELECT * FROM (
   SELECT
     cleabs,
     lien_vers_troncon_entree,
     -- liens_vers_troncon_sortie
     regexp_split_to_table(bduni_non_com_tmp.liens_vers_troncon_sortie, E'/') AS lien_vers_troncon_sortie
-  FROM bduni_non_com_tmp
-  WHERE lien_vers_troncon_entree IN (SELECT cleabs from edges) AND NOT gcms_detruit
+  FROM {output_schema}.bduni_non_com_tmp
```

Comment:
here, we should delete the output_schema. (sorry, but the generation time is very long at home...)